### PR TITLE
POI FINAL FIXES AND RESOLVING TEST CASES/dhruv

### PR DESCRIPTION
**Pull request by Dhruv-deriv**
SOLVED: 

1.Scroll issue in Proof of Identity 

Issue:
Screenshot 2024-06-10 at 9 59 41 AM

Fixed: 
Screenshot 2024-06-10 at 9 59 05 AM

2.Fixing failed test cases

### CODE REVIEW

File: packages/account/src/Components/financial-details/__tests__/financial-details.spec.tsx [hunk 1]
```diff
@@ -70,14 +70,13 @@ describe('<FinancialDetails />', () => {
...
-        (isDesktop as jest.Mock).mockReturnValue(false);
-        (isMobile as jest.Mock).mockReturnValue(true);
+        (useDevice as jest.Mock).mockReturnValueOnce({ isDesktop: false });
 
         renderComponent({});
 
         fieldsRenderCheck();
 
-        const inputs = screen.getAllByRole('combobox');
+        const inputs = screen.getAllByTestId('dt_dropdown_display');
```
Comment (amina-deriv) [inline packages/account/src/Components/financial-details/__tests__/financial-details.spec.tsx]: why do we change 'combobox check'

**Review by amina-deriv — APPROVED** [review]

File: packages/account/src/Components/personal-details/__tests__/personal-details.spec.tsx [hunk 2]
```diff
@@ -720,8 +713,8 @@ describe('<PersonalDetails/>', () => {
 
     it('should submit the form if there is no validation error on mobile', async () => {
         // [TODO] - Remove this when PersonalDetailsForm is migrated to TSX
-        (isMobile as jest.Mock).mockReturnValue(true);
-        (isDesktop as jest.Mock).mockReturnValue(false);
+        (useDevice as jest.Mock).mockReturnValueOnce({ isDesktop: false });
+        (useDevice as jest.Mock).mockReturnValueOnce({ isMobile: true });
```
Comment (utkarsha-deriv) [inline packages/account/src/Components/personal-details/__tests__/personal-details.spec.tsx]: We can remove the isMobile here

File: packages/core/src/sass/app/_common/components/account-common.scss [hunk 3]
```diff
@@ -754,7 +754,7 @@
 
         @include mobile-or-tablet-screen {
             overflow-y: scroll;
-            padding: 2.4rem 0 1.6rem;
```
Comment (utkarsha-deriv) [inline packages/core/src/sass/app/_common/components/account-common.scss:757]: we don't need 1.6 rem?

**Review by utkarsha-deriv — APPROVED** [review]

File: packages/core/src/sass/app/_common/components/account-common.scss (hunk 3 above)
Comment (Dhruv-deriv) [reply packages/core/src/sass/app/_common/components/account-common.scss:757]: not really we only need 2.4 rem  in all directions

**Review by utkarsha-deriv — APPROVED** [review]

**Pull request merged**